code/modules/mob/living/carbon/human/emote_vr.dm

```diff
@@ -23,14 +23,37 @@
 		if("mlem")
 			message = "mlems [get_visible_gender() == MALE ? "his" : get_visible_gender() == FEMALE ? "her" : "their"] tongue up over [get_visible_gender() == MALE ? "his" : get_visible_gender() == FEMALE ? "her" : "their"] nose. Mlem."
 			m_type = 1
+		if("blep")
+			message = "bleps [get_visible_gender() == MALE ? "his" : get_visible_gender() == FEMALE ? "her" : "their"] tongue out. Blep."
+			m_type = 1
 		if("awoo")
 			m_type = 2
 			message = "lets out an awoo."
 			playsound(loc, 'sound/voice/awoo.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("awoo2")
+			m_type = 2
+			message = "lets out an awoo."
+			playsound(loc, 'sound/voice/long_awoo.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("growl")
+			m_type = 2
+			message = "lets out a growl."
+			playsound(loc, 'sound/voice/growl.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("woof")
+			m_type = 2
+			message = "lets out an woof."
+			playsound(loc, 'sound/voice/woof.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("woof2")
+			m_type = 2
+			message = "lets out an woof."
+			playsound(loc, 'sound/voice/woof2.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
 		if("nya")
 			message = "lets out a nya."
 			m_type = 2
 			playsound(loc, 'sound/voice/nya.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("mrowl")
+			message = "mrowls."
+			m_type = 2
+			playsound(loc, 'sound/voice/mrow.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
```

Comment:
Whoops. That's what I get for looking at a white screen for too long. I think I'll leave it in though because I see both 'mrow' and 'mrowl' used a lot.